packages/Statie/src/Renderable/Latte/LatteFileDecorator.php

```diff
@@ -68,8 +68,9 @@ private function decorateFile(AbstractFile $file): void
         if ($file instanceof PostFile) {
             $parameters['post'] = $file;
 
-            // add "post" layout by default
-            $file->changeContent('{layout "post"}' . PHP_EOL . $file->getContent());
+            // add layout, "post" by default
+            $layout = $file->getLayout() ?: 'post';
+            $file->changeContent('{layout "' . $layout . '"}' . PHP_EOL . $file->getContent());
```

Comment:
Could you use `sprintf` on `'{layout "' . $layout . '"}'`?
I use it as convention here that is easier to read.